client/src/views/Hustenview/index.jsx

```diff
@@ -0,0 +1,68 @@
+import { hot } from 'react-hot-loader';
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+
+import { Header } from '../../components';
+import Hint from '../../components/Hint';
+import classify from './classif.js';
+
+const Title = () => (
+  <Typography component="p" variant="subtitle1" color="inherit">
+    Hast Du Husten?
+  </Typography>
+);
+
+const MyRadio = ({ options, id, question, preselected, info }) => {
+  const renderedoptions = options.map((opt, idx) => <FormControlLabel value={idx} control={<Radio />} label={opt} />);
+  const [value, setValue] = React.useState(preselected);
+  const handleChange = (event) => {
+    setValue(parseInt(event.target.value));
+  };
+  return (
+    <FormControl component="fieldset">
+      <FormLabel component="legend">{question}</FormLabel>
+      {info != undefined ? <Hint text={info} /> : <span />}
+      <RadioGroup name={id} value={value} onChange={handleChange}>
+        {renderedoptions}
+      </RadioGroup>
+    </FormControl>
+  );
+};
+
+const recordAndClassif = async function () {
+  alert(await classify());
```

Comment:
Do we have to put the answer in the store for the backend?

Reply:
By all means, but don't know how to do that. Is there any example where you put anything in store. I didn't find any back then.

Reply:
I've just looked again, there is/was a "addResult" that should use what is used in the store, but was nowhere called. Checked online in the dev branch, no change

Reply:
I have just grouped all state elements in the "Hustenview" and made so that the result of the classification smoothly flows into the same - the radio with ID husten.hustentyp is updated accordingly